bees/alertoverbee/README.md

```diff
@@ -1,14 +1,14 @@
-# AlertOverbee
+# AlertOver bee
```

Comment:
Other instances use "Bee", so you should capitalize here too.

Reply:
They were a mix. My quick inventory had more lower case. Happy to update for Title Case